cfme/tests/automate/test_method.py

```diff
@@ -797,3 +827,96 @@ def test_delete_tag_from_category(custom_instance):
                 "instance": instance.name,
             },
         )
+
+
+@pytest.mark.customer_scenario
+@pytest.mark.meta(automate=[1713072, 1745197])
+def test_automate_task_schedule(appliance, custom_automate_setup, current_server_time, request):
+    """
+    Polarion:
+        assignee: dgaikwad
+        initialEstimate: 1/8h
+        caseposneg: positive
+        casecomponent: Automate
+        setup:
+            1. Create domain, namespace, class and instance
+            2. Also create automate method with below ruby code:
+                >> $evm.log(:info, "Hello World")
+        testSteps:
+            1. Go to Configuration > Settings > Zones > Schedules
+            2. Create schedule with required fields:
+               >> Action - Automation Tasks
+               >> Object Details(Request) - Call_Instance
+               >> Attribute/Value Pairs
+                     >> domain - domain_name
+                     >> namespace - namespace_name
+                     >> class - class_name
+                     >> instance - instance_name
+               >> Timer Options
+            3. Check automation logs more than 1 times
+        expectedResults:
+            1.
+            2.
+            3. Automate method should be executed on scheduled time.
+
+    Bugzilla:
+        1713072
+    """
+    domain, namespace, auto_class, instance = custom_automate_setup
+    current_time, tz_num = current_server_time
+    start_date = current_time + relativedelta.relativedelta(minutes=5)
+    view = navigate_to(appliance.collections.system_schedules, 'Add')
+    available_list = view.form.time_zone.all_options
+
+    for tz in available_list:
+        if '{}:00'.format(tz_num[0:3]) in tz.text:
+            tz_select = tz.text
+            break
```

Comment:
if not match then what will be`tz_select`?

Reply:
changed code and added validation too